.github/workflows/screenshot.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ROLE_NON_PROD }}
+          role-to-assume: ${{ secrets.AWS_ROLE_SCREENSHOT }}
```

Comment:
I made this change too! so this may need a rebase

Reply:
Should be good now. I forget to force push after rebase and pull all these commit locally again. Now squashed into one commit.